### DISCUSSION TITLE
Github代码Code下重影

### DISCUSSION THREAD

**Discussion by qhq** [F9y4ng/GreasyFork-Scripts#191] [问答专区(Question / Answer)]
![QQ截图20230408220510](https://user-images.githubusercontent.com/3319385/230725499-790e4d78-5e47-49c6-aa7f-965b88945dbe.png)
右侧的    height: 1060px 去掉就正常

**Comment by F9y4ng** [comment]
下个更新会推送针对**Feature preview**中**New Code Search and Code View**新代码样式的修正，效果大致如下：

![fixed](https://user-images.githubusercontent.com/15855068/234301685-6156450c-30d2-45aa-9c29-525722e5f558.png)

**Comment by Xia0xia0Xia0** [comment]
这个效果可太棒了！比Github默认的选取样式好太多了！
----- 原始邮件 -----
发件人：F9y4ng ***@***.***>
收件人：F9y4ng/GreasyFork-Scripts ***@***.***>
抄送人：Subscribed ***@***.***>
主题：Re:_[F9y4ng/GreasyFork-Scripts]_Github代码Code下重影_(Discussion_#191)
日期：2023年04月25日 22点04分


下个更新会推送针对Feature preview中New Code Search and Code View新代码样式的修正，效果大致如下：


&mdash;
